src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -3031,6 +3031,44 @@ public void testTestWorkflowWithCronSchedule() {
     Assert.assertEquals("run 2", lastCompletionResult);
   }
 
+  public static class TestCronParentWorkflow implements TestWorkflow1 {
+
+    private final TestWorkflowWithCronSchedule cronChild =
+        Workflow.newChildWorkflowStub(TestWorkflowWithCronSchedule.class);
+
+    @Override
+    public String execute(String taskList) {
+      return cronChild.execute(taskList);
+    }
+  }
+
+  @Test
+  public void testChildWorkflowWithCronSchedule() {
+    // Min interval in cron is 1min. So we will not test it against real service in Jenkins.
+    // Feel free to uncomment the line below and test in local.
+    Assume.assumeFalse("skipping as test will timeout", useExternalService);
+
+    startWorkerFor(TestCronParentWorkflow.class, TestWorkflowWithCronScheduleImpl.class);
+
+    WorkflowStub client =
+        workflowClient.newUntypedWorkflowStub(
+            "TestWorkflow1::execute",
+            newWorkflowOptionsBuilder(taskList)
+                .setExecutionStartToCloseTimeout(Duration.ofHours(10))
+                .build());
+    registerDelayedCallback(Duration.ofHours(3), client::cancel);
```

Comment:
I prefer sleep after the start instead of the callback.

Reply:
Done.